pages/blog/designing_your_apis_with_asyncapi_part_1.md

```diff
@@ -6,7 +6,7 @@ tags:
   - tools
   - code generation
   - design 
-cover: /img/posts/jonaslagoni-miniseries-part1/blog-miniseries-cover.webp
+cover: /img/posts/jonaslagoni/miniseries-part1/blog-miniseries-cover.webp
```

Comment:
why do you want to have a dedicated, jonas-only folder for your articles?

Reply:
Thought it was easier to keep the related resources in one place 🤔

Reply:
we group them in folders that have name of the blog, this is how we group, like you have `/miniseries-part1/`

Reply:
Fair enough, I will re-organize back to the original 🙂